source/javascripts/modules/highlighted-nav.js

```diff
@@ -0,0 +1,47 @@
+(function($, Modules) {
+  'use strict';
+
+  Modules.HighlightedNav = function HighlightedNav() {
+    var $tocPane;
+    var $contentPane;
+
+    this.start = function start($element) {
+      $tocPane = $element.find('.app-pane__toc');
+      $contentPane = $element.find('.app-pane__content');
+      $contentPane.on('scroll', handleScroll);
```

Comment:
This could probably use some kind of debouncing, a la [lodash.debounce](https://lodash.com/docs/#debounce).